dojo/api_v2/serializers.py

```diff
@@ -1,6 +1,7 @@
 import json
 import logging
 import re
+from collections import UserList
```

Comment:
I just noticed that using `from collections import UserList` and referring to it as `UserList` isn't going to match [the PR in `dev`](https://github.com/DefectDojo/django-DefectDojo/pull/11224). Can you use the same approach from that PR to avoid potential merge conflicts?


```suggestion
import collections
```